app/scripts/app.js

```diff
@@ -1,5 +1,3 @@
-require('angular-bootstrap');
```

Comment:
This was not needed, and actually won't work unless you use something like Node or a Webpack. We can talk more about this later.